ComposeAdvanced/app/src/main/java/com/example/android/wearable/composeadvanced/presentation/MainActivity.kt

```diff
@@ -117,7 +117,7 @@ fun WearApp(watchRepository: WatchRepository) {
             currentBackStackEntry?.arguments?.getSerializable(SCROLL_TYPE_NAV_ARGUMENT)
                 ?: DestinationScrollType.NONE
 
-        var displayValue by remember { mutableStateOf(2) }
+        var displayValueForUserInput by remember { mutableStateOf(2) }
```

Comment:
It would be great if you could add a comment here as well... something like.. ?

// Display value is passed down to various user input screens, for the slider and stepper 
// components specifically, to demonstrate how they work.

Reply:
I added the comment, but this is temporary because I'll replace it with viewModel later.